### DISCUSSION TITLE
A favor from the developer please - I want to create a "birdseye"-like "camera" on a standalone go2rtc server, how can I do this on the go2rtc config?

### DISCUSSION THREAD

**Discussion by Majestic7979** [blakeblackshear/frigate#14445] [Ask A Question]
Hi Nick, I know you'll know this one (hoping!)

I have a standalone go2rtc I use for monitoring 3D Printers, so I have RTSP streams feeding into that, but currently if I want to cast any output to a TV, I can only cast a single camera at a time.

I would really love to get a "birdseye"-like view of these cameras on that install, so that I could stream just one source to a display monitor and have an overview of all my prints.

I am guessing you guys create the "birdseye" by using some particular go2rtc config?

What could I put on my go2rtc config yaml to achieve this, so that I could get a single stream with this "birdseye" of my 3D Printers? I know it is unrelated to Frigate, as what I want to do has nothing related to recording/detection... If you could tell me the config stanza to use that would be amazing!

Many thanks and congrats on this ever-evolving project, I love Frigate and have been using it since the beginning. You guys are awesome for creating such a fantastic solution for CCTV which is free and open source.#

Edit for more context:

Currently I get something quite similar by going to the ip:1984 (just got the Big brother reference and I've been using go2rtc for ages!)
Then, I select "Streams" at the top, untick everything apart from webrtc, and next to the Name I tick the box to select all cameras, then I click on stream and it opens a URL like this:

```
http://192.168.20.20:1984/stream.html?src=p1&src=p2&src=p3&src=p4&src=p5&mode=webrtc
```
On that page I can get a grid of all cameras playing simultaneous, they are loaded one by one, but I guess there may be a way to use a ffmpeg command to create a single grid of all cameras that I can play back via RTSP on VLC or something? I'm trying to find out what I should put on the config tab... under the "streams:" stanza.

**Comment by hawkeye217** [comment]
I'm not Nick, but I'm one of the other frigate devs. Nick may also have some thoughts, so I'll let him chime in when he can.

Birdseye isn't created using go2rtc, it actually is one of Frigate's older features and is created from the raw decoded pixel data of your detect streams.

What you're doing now with go2rtc's dashboard is probably the best solution. I don't think there's a simple or easy way to remux a bunch of RTSP streams into a single stream without decoding and math (much like birdseye is doing now).

**Reply by Majestic7979** [reply]
Hey, thanks for writing code for Frigate, you're awesome!

I was hoping there would be a way :(

It's not possible to stream the webpage as a video without some hacking, and while I love IT, coding is not my forte (due to neurological issues affecting math)...

If anyone knows of a way I'd be really grateful to know :)

If not, then thanks anyway.

**Comment by moandcompany** [comment]
This discussion on ffmpeg's `vstack` and `hstack` filters might be what you're looking for. Some usage examples:
https://stackoverflow.com/questions/11552565/vertically-or-horizontally-stack-mosaic-several-videos-using-ffmpeg 

I use go2rtc to transcode the existing Frigate birdseye stream to different bitrate targets and alternate codecs using some of the raw ffmpeg options that you can pass into go2rtc. Hopefully these filters can be used to work with go2rtc as well.
